components/gradle.properties

```diff
@@ -1,21 +1,26 @@
-org.gradle.jvmargs=-Xmx2048m -Dfile.encoding=UTF-8
+#Gradle
+org.gradle.jvmargs=-Xmx2048M -Dfile.encoding=UTF-8 -Dkotlin.daemon.jvm.options\="-Xmx2048M"
+org.gradle.caching=true
+org.gradle.configuration-cache=true
+
+#Android
 android.useAndroidX=true
-android.enableJetifier=true
-kotlin.code.style=official
-# __KOTLIN_COMPOSE_VERSION__
-kotlin.version=1.8.22
-# __LATEST_COMPOSE_RELEASE_VERSION__
-compose.version=1.5.0-dev1112
-agp.version=7.3.1
+
+#Versions
+kotlin.version=1.9.10
+compose.version=1.5.3
+agp.version=8.1.2
+
+#Compose
 org.jetbrains.compose.experimental.jscanvas.enabled=true
 org.jetbrains.compose.experimental.macos.enabled=true
 org.jetbrains.compose.experimental.uikit.enabled=true
+compose.desktop.verbose=true
 
+#Kotlin
+kotlin.code.style=official
+kotlin.mpp.enableCInteropCommonization=true
```

Comment:
We have a different properties now. Were each property changed consciously?

Reply:
Yes. I set the properties by intention and removed outdated ones

Reply:
By fact, it is required only for an IDE code completion. So, I'll roll back it

Reply:
As I understand this property also affects publication process.
But as I correctly understand previous problems was with SkikoUIView class compatibility.
So, for now - it should be fine!

Reply:
> it is required only for an IDE code completion

> was with SkikoUIView class compatibility.

Let's keep the property only if we are sure if we need it